.github/workflows/license-year-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Update copyright year in license files
+
+on:
+  schedule:
+    - cron: "0 3 1 1 *"
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions:checkout@v2
+        with:
+          fetch-dept: 0
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          path: ./**/*.kt
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          path: ./**/*.kts
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ./*.kts
```

Comment:
Why do we need these extra steps, doesn't the first one (without parameters) cover everything?..

Reply:
the firs one without parameters cover only `LICENSE` file

Reply:
Thanks, I see now:

```
    # A path or wildcard pattern specifying files to transform. Multiple paths can be
    # specified using literal styled YAML.
    #
    # Required: false
    # Default: LICENSE
    path: ''
```

Can we then use that "literal styled YAML" to have everything in one step? Here is an example:

https://github.com/FantasticFiasco/action-update-license-year#i-want-to-update-all-my-licenses-i-have-more-than-one
```
steps:
- uses: actions/checkout@v2
  with:
    fetch-depth: 0
- uses: FantasticFiasco/action-update-license-year@v2
  with:
    token: ${{ secrets.GITHUB_TOKEN }}
    path: |
      LICENSE-OPEN-SOURCE
      LICENSE-COMMERCIAL
```

Reply:
Ty. Tomorrow I'll try it in my test repository